build/install_ref.py

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env python3
+from SigProfilerMatrixGenerator import install as genInstall
+genInstall.install('GRCh37')
```

Comment:
does it only work with GRCh37? I'm thinking if the reference name can be an argument to the python script, so you can install other references easily in the future.

Reply:
Agree with @byb121, even if hardcoded in the Dockerfile the script should accept as an argument.